tkp/sourcefinder/extract.py

```diff
@@ -958,6 +958,11 @@ def _physical_coordinates(self):
         self.start_smin_y = (self.y.value - numpy.sin(self.theta.value) *
                       self.smin.value)
 
+        # Fix negative ra values going into the database as this messes
+        # up the source association step.
+        if self.ra<0.:
+            self.ra=self.ra+360.
```

Comment:
With apologies for the drive by reviewing — would `self.ra = self.ra % 360` be more general? It should do the right thing with RA 361°, -181°, etc...

Reply:
Drive by reviewing is completely fine - the more eyes the better :) 

That is a really neat solution, I didn't know that you could do that with %. I will incorporate it and run through my tests.